README.rst

```diff
@@ -90,6 +90,7 @@ TinyUSB is completely thread-safe by pushing all Interrupt Service Request (ISR)
 
 - **No OS**
 - **FreeRTOS**
+- **RT-Thread** (https://github.com/tfx2001/tinyusb)
```

Comment:
it should be linked to `https://github.com/RT-Thread/rt-thread`, though why would you close the PR ?

Reply:
there is a problem on my side. when i fix that, i will re create a pr.